android/api.md

```diff
@@ -594,6 +609,252 @@ private static String MyOIDCAuthProvider(){
 }
 ```
 
+#### Multi-Auth
+
+This section talks about the capability of AWS AppSync to configure multiple authorization modes for a single AWS AppSync endpoint and region. Follow the [AWS AppSync Multi-Auth](https://docs.aws.amazon.com/appsync/latest/devguide/security.html#using-additional-authorization-modes) to configure multiple authorization modes for your AWS AppSync endpoint.
+
+As discussed in the above linked documentation, certain fields may be protected by different authorization types. This can lead the same query, mutation, or subscription to have different responses based on the authorization sent with the request; therefore, it is recommended to use different `AWSAppSyncClient` objects for each authorization type. Instantiation of multiple `AWSAppSyncClient` objects is enabled by passing `true` to the `useClientDatabasePrefix` flag. The `awsconfiguration.json` generated by the AWS AppSync console and Amplify CLI will add an entry into the `AppSync` section called `ClientDatabasePrefix`. This will be used to separate the caches used for operations such as queries, mutations, and subscriptions.
```

Comment:
The `awsconfiguration.json` generated by the AWS AppSync console and Amplify CLI will add an entry into the `AppSync` section called `ClientDatabasePrefix` --> Is the section called "AppSync" or "ClientDatabasePrefix" ? I think it is called "AppSync" in which case we should say "... will add an entry called 'ClientDatabasePrefix" in the "AppSync" section"

Reply:
Changed.